frontend/src/utils/generate-github-auth-url.ts

```diff
@@ -5,6 +5,6 @@
  * @returns The URL to redirect to for GitHub OAuth
  */
 export const generateGitHubAuthUrl = (clientId: string, requestUrl: URL) => {
-  const redirectUri = `${requestUrl.origin}/oauth/github/callback`;
+  const redirectUri = `${requestUrl.origin}/oauth`;
```

Comment:
this is in our github settings and hard to change without causing an outage. Can we revert back to the old one?